packages/fx-core/src/plugins/resource/localdebug/index.ts

```diff
@@ -136,6 +136,11 @@ export class LocalDebugPlugin implements Plugin {
             EOL: os.EOL,
           }
         );
+
```

Comment:
Will this work for addResource?

Reply:
Yes. `addResource` will call `scaffold`.

Reply:
If users scaffold a tab project, and edit their vscode workspace settings (`.vscode/settings.json`), and then add resource. Will `addResource` overwrite the users changes to `.vscode/settings.json`

Reply:
Yes, and so is `.vscode/tasks.json` and `.vscode/launch.json`. I think it is an improvement that can be considered afterwards. It is easy to handle `.vscode/settings.json`, but hard to handle the other two.

Reply:
Sure. Thanks for the explanation.